### ISSUE TITLE
CUDA error while refining mesh。I configured cuda11.0，Are these errors due to version problems or?（细化网格时cuda报错）

### ISSUE THREAD

**Issue by xhb951120474** [cdcseacave/openMVS#713]
09:34:07 [App     ] Build date: Nov 10 2021, 09:13:11
09:34:07 [App     ] CPU: Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz (8 cores)
09:34:07 [App     ] RAM: 62.75GB Physical Memory 31.47GB Virtual Memory
09:34:07 [App     ] OS: Linux 5.4.0-73-generic (x86_64)
09:34:07 [App     ] SSE & AVX compatible CPU & OS detected
09:34:07 [App     ] CUDA device 0 initialized: GeForce RTX 2060 SUPER (compute capability 7.5; memory 7.79GB)

The above is my experimental machine, and the same error is reported on another machine with two 2080ti graphics cards
# 上面是我实验的机器，在另外一台两张2080ti显卡的机器上也是同样的报错

[App     ] CUDA error: view.depthMap.Reset(sizeof(float)*area) (CUDA_ERROR_OUT_OF_MEMORY (code 2) - out of memory)
 [App     ] CUDA error: view.baryMap.Reset(sizeof(hfloat)*3*area) (CUDA_ERROR_OUT_OF_MEMORY (code 2) - out of memory)
[App     ] CUDA error: view.depthMap.Reset(sizeof(float)*area) (CUDA_ERROR_OUT_OF_MEMORY (code 2) - out of memory)
[App     ] CUDA error: view.faceMap.Reset(sizeof(FIndex)*area) (CUDA_ERROR_OUT_OF_MEMORY (code 2) - out of memory)
 [App     ] CUDA error: view.baryMap.Reset(sizeof(hfloat)*3*area) (CUDA_ERROR_OUT_OF_MEMORY (code 2) - out of memory)
[App     ] CUDA error: view.depthMap.Reset(sizeof(float)*area) (CUDA_ERROR_OUT_OF_MEMORY (code 2) - out of memory)
[App     ] CUDA error: view.baryMap.Reset(sizeof(hfloat)*3*area) (CUDA_ERROR_OUT_OF_MEMORY (code 2) - out of memory)
And:
[App     ] CUDA error: kernelCrossCheckProjection(size, view.depthMap, view.faceMap, size.width, size.height ) (CUDA_ERROR_ILLEGAL_ADDRESS (code 700) - an illegal memory access was encountered)
 [App     ] CUDA error: cuMemsetD32(view.depthMap, (uint32_t&)fltMax, size.area()) (CUDA_ERROR_ILLEGAL_ADDRESS (code 700) - an illegal memory access was encountered)
 [App     ] CUDA error: kernelProjectMesh((int)faceIDsView.GetSize(), vertices, faces, faceIDsView, view.depthMap, view.faceMap, view.baryMap, CameraCUDA(camera, size), faceIDsView.GetSize() ) (CUDA_ERROR_ILLEGAL_ADDRESS (code 700) - an illegal memory access was encountered)

I configured cuda11.0，Are these errors due to version problems or?
# 我安装的是CUDA11.0，这些报错是版本的的问题还是其他的问题

How can I solve this problem？
# 我该如何去解决这个问题

**Comment by cdcseacave** [comment]
It run out of memory, pls use the CPU version for mesh refinement

**Comment by xhb951120474** [comment]
> 内存不足，请使用 CPU 版本进行网格细化

Yes, the CPU I use now is refining the grid, but the speed is very slow. If you want to use GPU acceleration, what are the requirements for hardware and CUDA versions?

**Comment by cdcseacave** [comment]
Closing as stale. The codebase has been substantially updated on the \develop\ branch. Please reopen with reproduction steps if the issue persists on the latest code.
